Documentation/report.md

```diff
@@ -297,3 +297,29 @@ A report can have the following states:
 
 
 [rfc3339]: https://tools.ietf.org/html/rfc3339#section-5.8
+
+## Roll-up Reports
+
+Report data is stored in the database much like metrics themselves, and can thus be used in aggregated or roll-up reports. A simple use case for a roll-up report is to spread processing load across a period of time: instead of running a monthly report all at once, it can instead be a simple sum of daily reports.
```

Comment:
`is to spread processing load` could potentially be more clear we're talking about the processing load to produce a report. Perhaps: ` is to spread the processing required to produce a report`? Not sure I like that either. I just think we need report/reporting in there some how.